.github/workflows/ping.yaml

```
@@ -0,0 +1,15 @@
+name: Keep Render Alive (${{ secrets.PING_ENABLED == 'true' && 'ENABLED' || 'DISABLED' }})
+
+on:
+  schedule:
+    - cron: '*/14 23-15 * * *'  # KST 08:00 ~ 24:00
```

actionlint (1.7.7)

invalid CRON format "*/14 23-15 * * *" in schedule event: beginning of range (23) beyond end of range (15): 23-15

(events)